### PR TITLE
[12.0][MIG] report_wkhtmltopdf_param

### PR DESCRIPTION
**Pull request by Abranes**
Standard migration to 12.0

cc @Jortolsa @fuentes010 @ioans73 @Saphery @Willerr-mo

### CODE REVIEW

**Comment by gdgellatly** [comment]
@Abranes there is no way to delete a custom setting logged in as admin user or any user for that matter. I don't think it should be necessary to Become SuperUser to delete a setting. Otherwise tested and works.

**Comment by gustavovalverde** [comment]
@Abranes tested and working. Could you make the fix exposed by @gdgellatly so we can proceed with some approval love? 😄

**Comment by andres-pcg** [comment]
@Abranes nice and tidy, could you make the change suggested by @gdgellatly? it would be nice to have checks printing functionality on v12. Thanks in advance.

**Comment by andres-pcg** [comment]
Any updates on this one ?

**Comment by Abranes** [comment]
Hello, sorry for the late response.
@gdgellatly I added a new access rule for Administration/Settings to give permission to them to delete a setting.

I don't know what happen with @oca-clabot because I signed the CLA, anyone can review?

**Comment by gdgellatly** [comment]
@Abranes don't worry about CLA. It happens due to historic commits on migrated modules.

File: report_wkhtmltopdf_param/security/ir.model.access.csv [hunk 1]
```diff
@@ -1,3 +1,4 @@
 "id","name","model_id:id","group_id:id","perm_read","perm_write","perm_create","perm_unlink"
 "paperformat_parameter_access_portal","ir_actions_report_paperformat_parameter group_portal","model_report_paperformat_parameter","base.group_portal",1,0,0,0
 "paperformat_parameter_access_employee","ir_actions_report_paperformat_parameter group_hr_user","model_report_paperformat_parameter",,1,0,1,0
+"paperformat_parameter_access_administration","ir_actions_report_paperformat_parameter group_system","model_report_paperformat_parameter","base.group_system",1,0,1,1
```
Comment (gdgellatly) [inline report_wkhtmltopdf_param/security/ir.model.access.csv]: @Abranes why no write rights here?

Comment (Abranes) [reply report_wkhtmltopdf_param/security/ir.model.access.csv]: @gdgellatly I forgot, now that group can do everything.

**Review by gdgellatly — APPROVED** [review]

**Comment by andres-pcg** [comment]
Hey so this is ready @pedrobaeza , thanks

**Review by gustavovalverde — APPROVED** [review]

**Pull request merged**